node/node.go

```diff
@@ -34,6 +35,7 @@ type Node struct {
 	Host        host.Host
 	ConnGater   connmgr.ConnectionGater
 	Routing     routing.PeerRouting
+	Exchange    exchange.Interface // TODO: Better name?
```

Comment:
Yes pls better name :)

Reply:
The problem here is that I am not sure about what name to use. The most descriptive would look like IPFSBlockExchange, but I don't like the IPFS word in it and if remove it, then it will be just BlockExchange, which does not really apply to us, as Blocks in IPFS and Blocks in our terms are different. 
Maybe DataExchange? For most of the data, we will have stored in IPFS blocks anyway.
There is also an option for ShareExchange but that's a little bit more than just that.

Reply:
I think `DataExchange` is fine for now.

Reply:
Agreed then

Reply:
It is now updated